src/middleware/applyMiddleware.ts

```diff
@@ -5,13 +5,13 @@ const finalMiddleware = (store: Store, args) => (action: Function) =>
   set(store, action(store.getState(), ...args))
 
 export default function applyMiddleware(...middlewares) {
+  middlewares = middlewares.reverse() // just reverse once
```

Comment:
@marshalYuan could you please remove the comment?

Reply:
👌